external/audiocodec/opus/opus_decoder_api.c

```diff
@@ -0,0 +1,169 @@
+/******************************************************************
```

Comment:
I think this can be moved to framework folder.
because we should maintain this

Reply:
ok, I will, how about `framework/src/media/codecs/`?
what's your suggestion?